hanoying_back/Tower[] towers # known towers
string[] floating_disks # known disks NOT on a known tower
geometry_msgs/Point[] floating_points # estimated position of those disks